### PR TITLE
Request of delivery

### PR DESCRIPTION
**Pull request by satorinic**
## PT-Link: https://www.pivotaltracker.com/story/show/150801863 ##
- added auto-mailing function for request of delivery to transportation 
- added an email from GigaFood containing delivery details

### CODE REVIEW

File: app/mailers/confirmation_mailer.rb [hunk 1]
```diff
@@ -14,4 +14,11 @@ def cancelation_email(order)
     @order.status = 'canceled'
     mail(to: @order.billing_email, subject: 'Cancelation from Gigafood')
   end
+
+  def move_by_bike_email(email, order)
+    @order = order
+    #@url = "http://example.com/movebybike"
+    #@order.status = 'delivery request'
```
Comment (tochman) [inline app/mailers/confirmation_mailer.rb]: Commented out code is not cool...

File: features/step_definitions/request_of_delivery.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  pending # Write code here that turns the phrase above into concrete actions
+end
+
+When(/^I click on "([^"]*)" for order "([^"]*)"$/) do |arg1, arg2|
+  pending # Write code here that turns the phrase above into concrete actions
+end
+
+When(/^I enter "([^"]*)" in "([^"]*)"$/) do |arg1, arg2|
+  pending # Write code here that turns the phrase above into concrete actions
+end
```
Comment (tochman) [inline features/step_definitions/request_of_delivery.rb]: Hmm... And this definitions? Not used? I would remove them.

**Review by tochman — CHANGES_REQUESTED** [review]

File: app/mailers/confirmation_mailer.rb (hunk 1 above)
Comment (satorinic) [reply app/mailers/confirmation_mailer.rb]: i'll remove it.
We need to set the status later

File: features/step_definitions/request_of_delivery.rb (hunk 2 above)
Comment (satorinic) [reply features/step_definitions/request_of_delivery.rb]: not used. Left over from previous testing. I delete the whole file

Comment (tochman) [reply features/step_definitions/request_of_delivery.rb]: It was added in this PR. But, yes, please remove.

Comment (satorinic) [reply features/step_definitions/request_of_delivery.rb]: i was created when we I started to work on the feature

**Review by tochman — APPROVED** [review]

**Pull request merged**